### DISCUSSION TITLE
How could I specify `taurui build` products?

### DISCUSSION THREAD

**Discussion by s3xysteak** [tauri-apps/tauri#11278] [Q&A] [answered]
I only want a `.exe` version. So I think it could be much more faster if I only build `.exe` instead of building for all default products.

**Comment by FabianLars** [comment, marked as answer]
`tauri build -b none` (v1) or `tauri build --no-bundle` (v2). Or you can change `bundle.targets` in tauri.conf.json accordingly.

**Comment by s3xysteak** [comment]
Thank you! You are the real hero.
